### PR TITLE
Update the dockerfile for Force to use the official node image

### PR DESCRIPTION
**Pull request by orta**
I was trying to get `hokusai dev start` to work to debug the dd-trace stuff, and had @bhoggard come over - he mentioned that we should consider moving off the bitnami image due to https://github.com/artsy/gravity/pull/11858 

So we moved force off, and switched to directly calling apt-get instead.

### CODE REVIEW

File: hokusai/development.yml [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
+      - NODE_ENV=development
+      - REDIS_URL=redis://force-redis:6379/0
+      - PORT=5000
+      - S3_KEY
+      - S3_SECRET
+      - DEPLOY_ENV
     env_file: ../.env
     ports:
-    - 5000:5000
-    command: npm start
```
Comment (orta) [inline hokusai/development.yml]: `command: npm start` got removed to make it consistent with staging/prod which don't have a command and fallback to `CMD`

**Comment by joeyAghion** [comment]
One thing to watch is the size of this new image. I believe we switched from the official node image to bitnami's only recently (https://github.com/artsy/force/pull/2352), though I don't know remember the thinking then. /cc @saolsen

File: Dockerfile [hunk 2]
```diff
@@ -1,17 +1,13 @@
...
-# 
-FROM bitnami/node:8.11.3
-#FROM circleci/node:8-stretch-browsers
+FROM node:8.11.3
 
-RUN install_packages libsecret-1-dev libglib2.0-dev
+# The key bits here are making sure we install, libsecret-1-dev libglib2.0-dev
+RUN apt-get update -qq && apt-get install -y \
+  libsecret-1-dev libglib2.0-dev && \
+  rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```
Comment (alloy) [inline Dockerfile]: What’s this for, to reduce the size of the resulting image?

File: src/lib/setup.js [hunk 3]
```diff
@@ -227,7 +227,11 @@ export default function(app) {
     // https://datadog.github.io/dd-trace-js/index.html
     // and we use all the defaults from:
     // https://github.com/DataDog/dd-trace-js/blob/master/docs/API.md#tracer-settings
-    ddTracer.init({})
+    console.log(Object.keys(process.env))
```
Comment (alloy) [inline src/lib/setup.js]: Stray loggy

**Comment by alloy** [comment]
Needs a rebase and do we need to wait re that question @joeyAghion ?

File: Dockerfile (hunk 2 above)
Comment (bhoggard) [reply Dockerfile]: Yes, it's typical to do it all in one `RUN` to reduce the size of that step's layer. It's what bitnami's `install_package` does.

Comment (alloy) [reply Dockerfile]: Thanks 👌

**Comment by orta** [comment]
Yeah, I think so - this isn't a blocker for any work, and it's worth waiting on @saolsen IMO

**Comment by alloy** [comment]
I’ll re-assign to him then.

**Comment by izakp** [comment]
Failed on a dependency install of `memwatch-ng` which is an OSX-specific dependency, oder?

**Comment by alloy** [comment]
> Failed on a dependency install of `memwatch-ng` which is an OSX-specific dependency, oder?

I don’t _think_ so, what makes you think that? https://github.com/srowatt/memwatch-ng#readme

At a glance, there appears to be V8 API related compile issues, maybe the API changed in this version of Node?

**Comment by izakp** [comment]
Oh oops - I was thinking of another library...

**Comment by izakp** [comment]
Looks like a transient build failure - re-ran and came up green

**Comment by alloy** [comment]
How weird… it definitely seemed to fail during compilation of C++ code :?

**Comment by saolsen** [comment]
Is there some problem with the bitnami image? I know we saw that gravity specifically had problems with it but lots of other services have use bitnami derived images and been fine. This doesn't seem to get us anything except a bigger image which slows down deploys.

**Comment by bhoggard** [comment]
I'm interested in testing performance with this image, since force has been relatively unstable and slow for the last few weeks. Could we try this image for a week and monitor it (or less if performance is worse)?

**Comment by saolsen** [comment]
alright

**Pull request merged**